css/bootstrap/bootstrap.css

```diff
@@ -2399,7 +2399,7 @@ table .span24 {
   float: left;
   min-width: 160px;
   padding: 4px 0;
-  margin: 1px 0 0;
+  /*margin: 1px 0 0;*/
```

Comment:
We need to remove code rather than commenting it out. If we'll need some code for future use, we can take it back from git history.